### DISCUSSION TITLE
the basic instruction isn't enough to make people install the app

### DISCUSSION THREAD

**Discussion by TCPuniverse** [Stirling-Tools/Stirling-PDF#1370] [General]
Hi.
The basic instructions isare very limited and for the people with docker/container skills, while it should be helpful for all people trying to use simple app for their daily use. I copied and paste the commands mentioned in the how-to guide, but it gave errors. some of them are:

Error: statfs /location/of/trainingData: no such file or directory
Error: statfs /location/of/extraConfigs: no such file or directory

there was nowhere in the doc for the folders that we need to create and the files we should initially create.

**Comment by MickGe** [comment]
Hi,

You can replace:
- `/location/of/trainingData` by `./trainingData`
- `/location/of/extraConfigs` by `./extraConfigs`

It will create directories. 

It's a convention. When you read `<change_me>` for a password, it means you should create your own password!

**Comment by TCPuniverse** [comment]
Hi.
still unsuccessful:

[user@localhost ~]$ podman images
REPOSITORY                TAG         IMAGE ID      CREATED      SIZE
docker.io/frooodle/s-pdf  latest      1361dab470ae  2 days ago   1.57 GB
docker.io/library/nginx   latest      2ac752d7aeb1  7 weeks ago  192 MB


[user@localhost ~]$ podman run -d \
  -p 8123:8080 \
  -v ./trainingData:/usr/share/tessdata \
  -v ./extraConfigs:/configs \
  -v ./logs:/logs \
  -e DOCKER_ENABLE_SECURITY=false \
  -e INSTALL_BOOK_AND_ADVANCED_HTML_OPS=true \
  -e LANGS=en_GB \
  --name stirling-pdf \
  1361dab470ae

Error: lstat trainingData: no such file or directory

Would somebody just mention which files and folders should be available in where so helping to ease the discovery burdon?!

**Reply by MickGe** [reply]
_Podman_ doesn't create directories.

As I read, you must create two directories… `./trainingData` and `./extraConfigs`.

When something doesn't work as expected, you can use a search engine. I don't really know _Podman_. I have used DuckDuckGo and found this: https://github.com/containers/podman/issues/10471.

On the documentation (https://github.com/containers/podman/blob/main/transfer.md), you could read:

> ## Behavioural differences and pitfalls
> 
> These commands behave differently from the commands in Docker:
> 
> | Command | Description |
> | :--- | :--- |
> | `podman volume create`                | While `docker volume create` is idempotent, `podman volume create` raises an error if the volume does already exist. See this [docker issue](https://github.com/moby/moby/issues/16068) for more information.|
> | `podman run -v /tmp/noexist:/tmp ...` | While `docker run -v /tmp/noexist:/tmp` will create non existing volumes on the host, Podman will report that the volume does not exist. The Podman team sees this as a bug in Docker.|
